src/3_PB/ManualeUtente/content.typ

```diff
@@ -0,0 +1,267 @@
+
+== Scopo del prodotto
+Lo scopo del prodotto è la realizzazione di un sistema di persistenza dati e successiva visualizzazione di questi, provenienti da sensori dislocati geograficamente. Tale piattaforma consentirà all'#glossary("amministratore pubblico") di acquisire una panoramica completa delle condizioni della città, facilitando così la presa di decisioni informate e tempestive riguardo alla gestione delle risorse e all'implementazione di servizi.
+
+== Glossario
+Al ﬁne di evitare possibili ambiguità relative al linguaggio utilizzato nei documenti, viene fornito il _Glossario v1.0_, nel quale sono presenti tutte le definizioni di termini aventi uno specifico significato che vuole essere disambiguato. Tali termini, sono scritti in  corsivo e marcati con una G a pedice.
+
+== Riferimenti
+=== Riferimenti normativi 
+- Capitolato C6 - InnovaCity: Smart city monitoring platform:
+#link("https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C6.pdf") (10/03/2024)
+
+- _Norme di Progetto v2.0_
+
+- Regolamento progetto didattico:
+#link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/PD2.pdf") (10/03/2024)
+
+=== Riferimenti informativi
+- _Analisi dei Requisiti v2.0_
+
+- _Specifiche Tecniche v1.0_
+
+- *#glossary[Grafana]*: #link("https://grafana.com/docs/grafana/latest/"); (10/03/2024)
+
+- *Docker*: #link("https://docs.docker.com/"); (10/03/2024)
+
+#pagebreak()
+
+= Requisiti minimi
+Per poter garantire un corretto funzionamento del prodotto, è necessario rispettare i requisiti minimi delineati in seguito.
+
+== Requisiti software
+I requisiti software indicano le specifiche relative al software che deve essere installato sul #glossary[sistema] dell'utente per garantire l'avvio e l'esecuzione del prodotto.
+
+#figure(
+table(
+      columns:(100pt,auto,auto),
+      align: (x, y) => (center, center, center).at(x),
+      fill:(_,row) => if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Software*],[*Versione*],[*Download*],
+      [Docker], [25.0.3], [#link("https://docs.docker.com/get-docker/")],
+      [#glossary[Docker Compose]], [2.24.6], [Integrato nel download di Docker]),
+caption: [Tabella requisiti software.])
+
+== Requisiti hardware
+I requisiti hardware indicano le specifiche tecniche relative alle componenti hardware necessarie per garantire un'esperienza di utilizzo ottimale del prodotto e prestazioni elevate.
+
+#figure(
+table(
+      columns:(100pt,auto),
+      align: (x, y) => (center, center).at(x),
+      fill:(_,row) => if row==0 {luma(150)} else if calc.odd(row) { luma(220)} else {white},
+      [*Componente hardware*],[*Requisito*],
+      [CPU], [Quad-Core 2,5 GHz],
+      [RAM], [6GB]),
+caption: [Tabella requisiti hardware.])
+
+== Requisiti di #glossary[sistema] operativo
```

Comment:
Requisiti di sistema operativo e di hardware come li hai ragionati? (solo per sapere, non per contestare)

Reply:
sinceramete non li ho particolarmente ragionati, ho semplicemente riportato quel che c'era già nell'AdR per il momento (a parte CPU e RAM che abbiamo discusso brevemente in uno stand-up); per il sistema operativo probabilmente toglierei Mint

Reply:
Propongo un MacOS, Windows, Ubuntu. Per quanto riguarda alla scelta dell'hardware dovremmo riuscire a motivarla perché se il prof ci chiede il come mai questi requisiti non possiamo fare scena muta.

Reply:
penso che saremo in grado di affinarla/motivarla una volta ottenuti i risultati dei test di performance, per il momento lascio così.